src/Explorer/Tables/TableDataClient.ts

```diff
@@ -208,6 +208,9 @@ export class CassandraAPIDataClient extends TableDataClient {
           !originalDocument[property] ||
           newEntity[property]._.toString() !== originalDocument[property]._.toString()
         ) {
+          if (newEntity[property]._.toString() === "" || newEntity[property]._.toString() === undefined) {
```

Comment:
I believe that undefined.ToString() will produce "undefined". Maybe change that second part of the clause to newEntity[property]._ === undefined ?

Reply:
removed .toString()